data/remnant/remnant outfits.txt

```diff
@@ -476,12 +477,19 @@ outfit "Anvil-Class Engine"
 	"thrust" 13.5
 	"thrusting energy" 2.05
 	"thrusting heat" 1.95
-	"turn" 336.0
+	"turn" 352.8
 	"turning energy" 1.05
 	"turning heat" 1.1
+	"reverse thrust" 8.1
+	"reverse thrusting energy" 1.23
+	"reverse thrusting heat" 1.17
```

Comment:
You're still missing the reverse flares and sounds.
And I'm still not interested in slowing resistance on these engines.

Reply:
Sorry, for some reason forgot that. Will be done momentarily.

The Remnant are paranoid about being able to escape. They got slow generating weapons from other aliens. If they can do it, best to assume that others can too. Either way, the amount of resistance is small but still enough to give a surprise edge in escaping from things using slow weapons against them. (0.04 for the smelter)